pkg/daemon/handler.go

```diff
@@ -65,10 +65,10 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	switch m.Action {
 	case "pause":
-		h.Logger.Info("pause request received, freezing pod...")
+		h.Logger.Info("pause request received, freezing pod: %s", resp.Status.User.Extra["authentication.kubernetes.io/pod-name"][0])
 		h.Freezer.Freeze(r.Context(), resp.Status.User.Extra["authentication.kubernetes.io/pod-uid"][0], "user-container")
 	case "resume":
-		h.Logger.Info("resume request received, thawing pod...")
+		h.Logger.Info("resume request received, thawing pod: %s", resp.Status.User.Extra["authentication.kubernetes.io/pod-name"][0])
```

Comment:
Let's pull out `resp.Status.User.Extra["authentication.kubernetes.io/pod-name"][0]` to a variable above the switch since it's kind of long to have copy/pasted four times

Reply:
I just realised we're logging pod-name but then freezing with pod-uid, shouldn't we log the thing we actually use?